src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -453,35 +470,41 @@ await using (var statisticsDbContext = InMemoryPublicStatisticsDbContext(publicS
                 .Returns(Task.CompletedTask);
 
             await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
+            await using (var statisticsDbContext = InMemoryStatisticsDbContext(publicStatisticsDbContextId))
```

Comment:
Different identifier here for `statisticsDbContext `? Also on L487.

Reply:
Done ta